desktop_version/src/Graphics.cpp

```diff
@@ -3469,6 +3472,54 @@ void Graphics::updatescreenshake(void)
     screenshake_y =  static_cast<Sint32>((fRandom() * 7) - 4);
 }
 
+void Graphics::draw_window_background(void)
+{
+    clear();
+}
+
+void Graphics::get_stretch_info(SDL_Rect* rect)
+{
+    int width;
+    int height;
+    gameScreen.GetScreenSize(&width, &height);
+
+    switch (gameScreen.scalingMode)
+    {
+    case SCALING_INTEGER:
+    {
+        int scale = SDL_min(width / SCREEN_WIDTH_PIXELS, height / SCREEN_HEIGHT_PIXELS);
+        rect->x = (width - SCREEN_WIDTH_PIXELS * scale) / 2;
+        rect->y = (height - SCREEN_HEIGHT_PIXELS * scale) / 2;
+        rect->w = SCREEN_WIDTH_PIXELS * scale;
+        rect->h = SCREEN_HEIGHT_PIXELS * scale;
+    }
+    break;
```

Comment:
The `break;` should be inside the block.
```suggestion
        break;
    }
```